Developer-Guide.md

```diff
@@ -569,6 +570,52 @@ file exists and contains details of the image and how it was created:
 $ cat /var/lib/osbuilder/osbuilder.yaml
 ```
 
+## Capturing kernel boot logs
+
+Sometimes it is useful to capture the kernel boot messages from a Kata Container
+launch. If the container launches to the point whereby you can `exec` into it, and
+if the container has the necessary components installed, often you can use `dmesg` at
+a shell prompt inside the container to view the kernel boot logs.
+
+If however you are unable to `exec` into the container, you can enable some debug
+options to have the kernel boot messages logged into the system journal.
+
+To achieve this you need to `enable_debug=true` in the Kata Configuration file
+for both the hypervisor/kernel and the proxy configuration sections. The following
+shows the differences as required in the `configuration-qemu.toml` file for instance:
+
+```patch
+@@ -194,7 +194,7 @@ enable_iothreads = @DEFENABLEIOTHREADS@
+ # to the proxy logs, but only when proxy debug is also enabled.
+ #
+ # Default false
+-#enable_debug = true
++enable_debug = true
+
+ # Disable the customizations done in the runtime when it detects
+ # that it is running on top a VMM. This will result in the runtime
+@@ -305,7 +305,7 @@ path = "@PROXYPATH@"
+
+ # If enabled, proxy messages will be sent to the system log
+ # (default: disabled)
+-#enable_debug = true
++enable_debug = true
+
+ [shim.@PROJECT_TYPE@]
+ path = "@SHIMPATH@"
+```
+
+The kernel boot messages will appear in the `kata-proxy` log such as:
+
+```bash
+$ journalctl -t kata-proxy
```

Comment:
in some systems `sudo` is required

Reply:
ack. Originally I tried to prefix with a root `#` prompt, but not only might that be too subtle, but it rendered as a comment! - updated.